librad/tests/working_copy.rs

```diff
@@ -204,6 +208,13 @@ async fn can_fetch() {
                 remote.fetch(&[&name], None, None).unwrap();
             }
         }
+        for res in local_transport_results_peer2
```

Comment:
So this list of results will be an exact correspondence to the `remote.fetch` actions above? Which means that like parentheses balancing, we need to ensure that every `git2` action has a corresponding `wait`?

Reply:
No unfortunately not -- `fetch` is a compound action:

> Convenience function to connect to a remote, download the data, disconnect and update the remote-tracking branches.

One `fetch` seems to correspond to three calls to `SmartSubtransport::action` -- which doesn't make sense, as one would expect two, but :man_shrugging:

Reply:
Even more terrifying :scream: